gensrc/script/functions.py

```diff
@@ -894,4 +894,12 @@
 
     # struct functions
     [170500, 'row', 'ANY_STRUCT', ['ANY_ELEMENT', "..."], 'StructFunctions::row'],
+
+    # unix timestamp extended version to int64
+    # [170700, 'unix_timestamp', 'BIGINT', [], 'TimeFunctions::to_unix_for_now_64'],
+    # [170701, 'unix_timestamp', 'BIGINT', ['DATETIME'], 'TimeFunctions::to_unix_from_datetime_64'],
+    # [170702, 'unix_timestamp', 'BIGINT', ['DATE'], 'TimeFunctions::to_unix_from_date_64'],
+    # [170701, 'unix_timestamp', 'BIGINT', ['VARCHAR', 'VARCHAR'], 'TimeFunctions::to_unix_from_datetime_with_format_64'],
+    [170704, 'from_unixtime', 'VARCHAR', ['BIGINT'], 'TimeFunctions::from_unix_to_datetime_64'],
+    # [170705, 'from_unixtime', 'VARCHAR', ['BIGINT', 'VARCHAR'], 'TimeFunctions::from_unix_to_datetime_with_format_64', 'TimeFunctions::from_unix_prepare', 'TimeFunctions::from_unix_close'],
```

Comment:
drop comment

Reply:
I just fix this function first. If you agree this change, I will fix other functions.

Reply:
if moved to line:400, then id is not contiguous, which I doubt that it will confuse people too.